### PR TITLE
added page

### PR DESCRIPTION
**Pull request by Kwang8**
Added a new page for Sproul.club, basically just followed this:

https://www.google.com/search?q=web+view+xcode&oq=web+view+xcode&aqs=chrome..69i57j0i10l9.1773j0j9&sourceid=chrome&ie=UTF-8

### CODE REVIEW

**Review by huangshawn1 — CHANGES_REQUESTED** [review]
It looks like the SproulClub folder isn't actually a group in the project (so I wasn't initially able to build the app). You want to add the folder as a group to the project (you should see the folder in the left project navigator in Xcode).

We want some spacing between the web view and the top/left/right/bottom elements. Typically, we use 6, 10, or 16 in other parts of the app but see what looks good.
It'd also be nice to disable the vertical scrolling indicator (we do that throughout the app) if possible. And disabling the pinch to zoom out would also be nice.

**Review by Kwang8 — COMMENTED** [review]
Moved everything into viewDidLoad. Added some constraints to the top, bottom, left, and right anchors. Disabled the vertical scroll indicator and pinch gestures.

**Comment by huangshawn1** [comment]
looks like the pinch gesture is still possible, I left a video on the Notion task: https://www.notion.so/berkeleymobile/Sprint-Board-iOS-3a44a1f5f7044adab2ea4b7827e7ea5c?p=60ca72b99d2b414fb1a743147d771258
take another look, but if it's too difficult to disable that's fine.

forgot to mention this, but can we also move this resource towards the front of the list? having it as the 2nd one (right after Dashboard) would be good.

File: berkeley-mobile/Resources/SproulClub/SproulClubViewController.swift [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            webView.translatesAutoresizingMaskIntoConstraints = false
+            self.view.addSubview(webView)
+            webView.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor, constant: 6).isActive = true
+            webView.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor, constant: 6).isActive = true
+            webView.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor, constant: -6).isActive = true
+            webView.bottomAnchor.constraint(equalTo: view.layoutMarginsGuide.bottomAnchor, constant: -20).isActive = true
+            
+            let myURL = URL(string:"https://www.sproul.club/")
+            let myRequest = URLRequest(url: myURL!)
+            webView.load(myRequest)
```
Comment (huangshawn1) [inline berkeley-mobile/Resources/SproulClub/SproulClubViewController.swift:32]: nitpick: change variable names (don't use my...)

File: berkeley-mobile/Resources/SproulClub/SproulClubViewController.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import WebKit
+
+class SproulClubViewController: UIViewController, WKUIDelegate, UIScrollViewDelegate {
+    var webView: WKWebView!
+        
+        override func viewDidLoad() {
+            super.viewDidLoad()
+            let webConfiguration = WKWebViewConfiguration()
+            webView = WKWebView(frame: .zero, configuration: webConfiguration)
+            webView.uiDelegate = self
```
Comment (huangshawn1) [inline berkeley-mobile/Resources/SproulClub/SproulClubViewController.swift:20]: I think the indent for viewDidLoad is off by 1

**Review by huangshawn1 — APPROVED** [review]
looks good

**Pull request merged**